go/service/identify.go

```diff
@@ -63,10 +63,12 @@ func (h *IdentifyHandler) Identify(_ context.Context, arg keybase1.IdentifyArg)
 	}
 
 	exp := res.Export()
-	if err := h.resultCache.Set(arg.UserAssertion, exp); err != nil {
-		h.G().Log.Debug("Identify: result cache set error: %s", err)
-	} else {
-		h.G().Log.Debug("Identify: storing result for %q in result cache", arg.UserAssertion)
+	if len(arg.UserAssertion) > 0 {
```

Comment:
don't want to cache empty string -> a user.
if they change users, will be incorrect.
doesn't really matter, because kbfs never id's empty user, and cache results only used for kbfs, but just to be safe.